core/templates/dev/head/components/profile_link.html

```diff
@@ -0,0 +1,8 @@
+<script type="text/ng-template" id="components/profileLinkText">
+	<a ng-if="!(linkedName === 'admin') && !(linkedName === 'OppiaMigrationBot')" ng-href="/profile/<[linkedName]>"><[linkedName]></a>
+	<span ng-if="linkedName === 'admin' || linkedName === 'OppiaMigrationBot'"><[linkedName]></span>
+</script>
+
+<script type="text/ng-template" id="components/profileLinkImage">
+	<a ng-href="/profile/<[linkedName]>"><ng-transclude></ng-transclude></a>
```

Comment:
I don't think you need the ng-transclude directive. Besides, shouldn't there be an image passed in?

I would have expected to see

    <a ng-href="/profile/<[linkedName]>">
      <img ng-src="<[profileImageDataUrl]>">
    </a>